charts/gardener-extension-networking-calico/templates/clusterparsers-logging.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: fluentbit.fluent.io/v1alpha2
+kind: ClusterParser
+metadata:
+  name: caliconode-parser
```

Comment:
In your case it can be as you have only one container, but most of the extensions has multiple containers, and the containers are using a different log format which means that we have to create a different filters which match the specific pod-container combination and then to apply the correct parser for them. That's why we choose to use this naming convention for   the filters

Reply:
Still, I do not fully understand why we omit the `-` here (as in `caliconode` vs. `calico-node`) while we use them in the other case. Could you please use them consistently?

Reply:
Yes, for this I agree that is is a typo and can be changed to calico-node

Reply:
Thanks.

Reply:
Addressed in the force-push